utils/utils.go

```diff
@@ -0,0 +1,27 @@
+package utils
```

Comment:
Since these utils are only used from tests, you can name this file `utils_test.go` so that this file is only even attempted to be compiled during tests